pkgs/tools/package-management/auditwheel/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+, patchelf
+, pbr
+, pretend
+, pyelftools
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonApplication rec {
+  pname = "auditwheel";
+  version = "4.0.0";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "03a079fe273f42336acdb5953ff5ce7578f93ca6a832b16c835fe337a1e2bd4a";
+  };
+
+  nativeBuildInputs = [
+    pbr
+  ];
+
+  propagatedBuildInputs = [
+    pyelftools
+  ];
+
+  # integration tests require docker and networking
+  disabledTestPaths = [ "tests/integration" ];
+
+  checkInputs = [
+    pretend
+    pytestCheckHook
+  ];
+
+  makeWrapperArgs = [
+    ''--prefix PATH : ${patchelf}/bin''
```

Comment:
```suggestion
    "--prefix PATH : ${lib.makeBinPath [ patchelf ]}"
```

Reply:
Is this just syntactic sugar or does it solve a problem?

Reply:
Actually `"--prefix" "PATH" ":" (lib.makeBinPath [ patchelf ])` would be even better.